src/app/ad-authoring/generate-amp-html.ts

```diff
@@ -0,0 +1,202 @@
+export function generateAdAmpHtml(
```

Comment:
Is there a particular reason why you switched from having common amphtml components and merging with video or image depending on the asset, to separate strings?

Reply:
In the downloadZip branch I switched it because I wanted to avoid completely rewriting adAMPHTML when the asset is a video since there is only one difference it the html, which is replacing <amp-img> tag with <amp-video> tag.